drake/solvers/MathematicalProgram.cpp

```diff
@@ -106,9 +115,10 @@ class NonlinearProgram : public MathematicalProgram {
     MixedIntegerQuadraticProgram {};
     struct LinearProgram : public QuadraticProgram, public
     MixedIntegerLinearProgram {
-    virtual MathematicalProgramInterface* addLinearEqualityConstraint() { return new
+    virtual MathematicalProgramInterface* AddLinearEqualityConstraint() { return
+   new
     LinearProgram; };
```

Comment:
Is this actually want clang-format does?  Ick.
Is it worth trying to `// BR` on prior lines here to choose where wrapping happens?